### PR TITLE
feat: add pull request template

### PR DESCRIPTION
**Pull request by thiti-y**
# Description

Add pull request template

# Related Issue

- Resolve https://github.com/Naga-DAO/naga-mint-workspace/issues/2

### CODE REVIEW

**Review by circles-ingots — APPROVED** [review]

**Review by circles-ingots — CHANGES_REQUESTED** [review]
test

**Review by circles-ingots — APPROVED** [review]

**Review by circles-ingots — APPROVED** [review]

**Review by circles-ingots — CHANGES_REQUESTED** [review]
test

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Description
+
+<!--- Your pull request description... -->
```
Comment (circles-ingots) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:3]: test comment

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+# Related Issue
```
Comment (circles-ingots) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:5]: ttttttttttttt

**Review by circles-ingots — CHANGES_REQUESTED** [review]

**Review by circles-ingots — APPROVED** [review]

**Review by thiti-y — COMMENTED** [review]
test

**Comment by thiti-y** [comment]
ppppp

**Review by circles-ingots — COMMENTED** [review]
fffffff

**Comment by circles-ingots** [comment]
eeeeee

**Review by circles-ingots — APPROVED** [review]

**Review by circles-ingots — CHANGES_REQUESTED** [review]
lllllllll

**Pull request merged**